<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QueryTab</class>
 <widget class="QWidget" name="QueryTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1159</width>
    <height>914</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="CodeEditor" name="codeEditor"/>
       </item>
      </layout>
     </widget>
     <widget class="QTabWidget" name="resultsTabBar">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Results</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="TableView" name="resultsGrid">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="dragDropOverwriteMode">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="exportResultsToFileButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="toolTip">
               <string>Export Results to File</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../resources/icons.qrc">
                <normaloff>:/icons/silk/icons/silk/page_white_go.png</normaloff>:/icons/silk/icons/silk/page_white_go.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="exportResultsToClipboard">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="toolTip">
               <string>Export Results to Clipboard</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../resources/icons.qrc">
                <normaloff>:/icons/silk/icons/silk/page_white_copy.png</normaloff>:/icons/silk/icons/silk/page_white_copy.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Messages</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QPlainTextEdit" name="resultsText"/>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CodeEditor</class>
   <extends>QPlainTextEdit</extends>
   <header>ui/CodeEditor.h</header>
  </customwidget>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>ui/TableView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/icons.qrc"/>
 </resources>
 <connections/>
</ui>
